### DISCUSSION TITLE
The "@current" filter in beta projects dashboard is not working.

### DISCUSSION THREAD

**Discussion by marcotas** [community/community#18834] [Projects and Issues] [answered]
When I select the filter current it doesn't show all my issues selected for that iteration (or sprint). I wonder if could be because today is the last day of the iteration and it could be a problem with timezone? If I move them to the next iteration they appear, but doesn't appear for the current. If I filter by the iteration it self it appears OK, but when I select the "current" it doesn't work, it disappears all of my issues for the current iteration.

**Comment by siemovit** [comment]
Hello marcotas, the same thing happened to me yesterday. Is it still the case for you ?

**Comment by galah92** [comment]
Same happens for me today.

**Comment by NevilleS** [comment]
I'm seeing this as well - the `@current` filter works as expected most of the time, but on the last day of an iteration it acts as if the sprint has ended. However, when you "group by iteration" in the table view, you'll see that the iteration is still tagged as `Current` there.

This is rather inconvenient, since Projects Beta doesn't allow you to overlap the start/end dates for iterations, so it seems like you're always left with one day where there is no `@current` iteration if you use the board view.

**Comment by NevilleS** [comment]
See https://github.com/github-community/community/discussions/20032 for more details on this

**Comment by 2percentsilk** [comment]
Investigating! Hoping to share more details with y'all soon - appreciate the flag on this issue.

**Comment by Neudrino** [comment]
There are several issues describing the same problem.

The **most up-voted** one is:
- https://github.com/github-community/community/discussions/20101

What was thought to be a **fix** was mentioned to be out for testing in:
- https://github.com/github-community/community/discussions/20101#discussioncomment-3149906

The others are:
- https://github.com/github-community/community/discussions/20217
- https://github.com/github-community/community/discussions/20032
- https://github.com/github-community/community/discussions/20108
- https://github.com/github-community/community/discussions/18834

**Comment by 2percentsilk** [comment, marked as answer]
Based on testing from the community and team, we believe we have fixed this bug. Thanks to everyone for reporting it!
